pkg/crc/cluster/kubeadminPassword.go

```diff
@@ -0,0 +1,95 @@
+package cluster
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"math/big"
+
+	"github.com/code-ready/crc/pkg/crc/machine/bundle"
+	"github.com/code-ready/crc/pkg/crc/oc"
+	"golang.org/x/crypto/bcrypt"
+)
+
+// UpdateKubeAdminUserPassword does following
+// - Generate a random password for kubeadmin user
+// - Update the kubeadmin password file
+// - Update the htpasswd secret
+func UpdateKubeAdminUserPassword(kubeAdminPassword string, ocConfig oc.Config, bundle *bundle.CrcBundleInfo) error {
+	// In case of cluster started from stopped state.
+	if kubeAdminPassword == "" {
+		return nil
+	}
+	hashDeveloperPasswd, err := hashBcrypt("developer")
```

Comment:
developer user is sudoer. I guess we should also rotate the password. WDYT?

Reply:
we can remove developer from sudoer.

Reply:
Yes but we need to also to update the doc to remove this kind of commands: `oc get co --as system:admin`.